openupgrade_scripts/scripts/hr_expense/16.0.2.0/post-migration.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2023 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def try_delete_noupdate_records(env):
+    openupgrade.delete_records_safely_by_xml_id(
+        env,
+        [
+            "hr_expense.product_product_fixed_cost",
+        ],
+    )
```

Comment:
What's the purpose of deleting this record from the DB? And what should I do if this product was used (for example in account move lines)? 

Up to now I get the following (non blocking) error: 
```
2023-11-29 15:03:43,202 48396 ERROR ou-odna-16-mig odoo.sql_db: bad query: DELETE FROM "product_product" WHERE id IN (157)
ERROR: update or delete on table "product_product" violates foreign key constraint "account_move_line_product_id_fkey" on table "account_move_line"
DETAIL:  Key (id)=(157) is still referenced from table "account_move_line".
```

Reply:
Sorry, you're right, this is not necessary. Checking again, I have added a `rename_xmlids` of a same product in v15 and v16.